app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema_db.db3")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            "(first_name, last_name) VALUES (?, ?)",
+            (
+                first_name,
+                last_name,
+            ),
+        )
+        self._connection.commit()
+
+    def all(self) -> list:
+        cinema_data_cursor = self._connection.execute(
+            "SELECT id, first_name," " last_name FROM actors"
```

Comment:
First of all why you need this " "
```suggestion
            "SELECT id, first_name, last_name FROM actors"
```
Also use select * for select all columns